src/main/java/data_access/UserDataAccessObject.java

```diff
@@ -19,15 +19,16 @@
 public class UserDataAccessObject implements ClubSignupUserDataAccessInterface, StudentSignupUserDataAccessInterface,
```

Comment:
Do we need JavaDocs for the functions? they seem to be missing.

Reply:
Just added them. Thanks for the reminder!